deploy/examples/cluster-multus-test.yaml

```
@@ -28,6 +28,12 @@ spec:
     selectors:
       public: public-net
       cluster: cluster-net
+    addressRanges:
+      public:
+        - "192.168.100.0/24"
+        - "192.168.101.0/24"
+      cluster:
+        - "192.168.200.0/24"
```

Does this work as well, if the NetAttDefs aren't modified? I would expect this to also work.

```suggestion
    addressRanges:
      public:
        - "192.168.20.0/24" # used for pods via NetAttDef
        - "192.168.25.0/24" # not used for pods, but Ceph won't break if this is added
      cluster:
        - "192.168.21.0/24"
```

---

done

let me know if we are planning to merge this, I'll mark is ready to merge

---

@BlaineEXE ^

---

In that case we can close this PR?

---

closing this one then